.travis.yml

```diff
@@ -11,35 +26,47 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=3.2.*
+  matrix: SYMFONY_VERSION=3.4
   global:
     - SYMFONY_DEPRECATIONS_HELPER=48
     - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml" SYMFONY_PHPUNIT_VERSION=5.7
+    - TEST_INSTALLATION=false
 
 matrix:
   include:
     - php: 7.1
+      env: SYMFONY_VERSION=3.4.*
+    - php: 7.1
       env: DEPS=dev SYMFONY_VERSION=3.3.*
-    - php: 5.6
+    - php: 7.1
       env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 7.1
       env: SYMFONY_VERSION=3.1.*
-    - php: hhvm
-      dist: trusty
+    - php: 7.1
+      env: SYMFONY_VERSION=3.2.*
+
+    - env: TEST_INSTALLATION=true
   fast_finish: true
+  allowed_failures:
+    - php: 7.1
+      env: SYMFONY_VERSION=3.4.*
+    - env: TEST_INSTALLATION=true
+
```

Comment:
@dbu can you give me a hand with the `allowed_failures` here? I know we do not like it. but i would like to see future states here, but i it does not really work the way i do it: https://travis-ci.org/symfony-cmf/seo-bundle

Reply:
ups, i should write it the right way